src/test/java/mate/academy/dao/impl/UserDaoImplTest.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.dao.impl;
+
+import mate.academy.dao.UserDao;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class UserDaoImplTest extends AbstractTest {
+    private UserDao userDao;
+    private User bob;
```

Comment:
you use it in one test method, make it local. Look through other classes and make local variables from fields if it's possible